contracts/extensions/Bank.sol

```diff
@@ -64,7 +64,7 @@ contract BankExtension is DaoConstants, AdapterGuard, IExtension {
     struct Checkpoint {
         // A checkpoint for marking number of votes from a given block
         uint96 fromBlock;
-        uint160 amount;
+        uint64 amount;
```

Comment:
that seems low to me. are you sure the recommendation was not to limit to 10^64 instead of 2^64. 
I've done a few calculations and 2^64 =~  2 * 10^19

10 ^19 wei is 10 ETH ... so def too low.
2^160 =~ 1.5 * 10 ^48 

that is 10^30 ETH which is a much better limit.

Reply:
internal tokens limited to 2^64
external tokens limited to 2^88
as we discussed on slack